app/models/destroy_question_option_modal.rb

```diff
@@ -0,0 +1,18 @@
+class DestroyQuestionOptionModal
+  include ActiveModel::Model
+  attr_accessor :service, :page, :question, :option, :label
+
+  delegate :expressions, :conditionals, to: :service
+
+  def to_partial_path
+    return 'api/question_options/cannot_delete_modal' if can_not_be_deleted?
+
+    'api/question_options/destroy_message_modal'
+  end
+
+  private
+
+  def can_not_be_deleted?
+    option && expressions.map(&:field).include?(option.uuid)
```

Comment:
```suggestion
    option.present? && expressions.map(&:field).include?(option.uuid)
```

minor

Reply:
How odd - I added this change during our call.   Apparently I failed to save it

Reply:
Ah no, I see - same change, different file.